corehq/apps/domain/models.py

```diff
@@ -164,7 +166,16 @@ class InternalProperties(DocumentSchema, UpdatableSchema):
     goal_followup_rate = DecimalProperty()
     # intentionally different from and commtrack_enabled so that FMs can change
     commtrack_domain = BooleanProperty()
-    performance_threshold = IntegerProperty()
+    performance_threshold = IntegerProperty(default=15)
+    experienced_threshold = IntegerProperty(default=3)
```

Comment:
i think this one is still pending. agree we'd probably want to be able to change defaults via constant and not save it to the DB. don't think it's hugely blocking though so will go ahead an merge